python/pyspark/mllib/util.py

```diff
@@ -169,6 +169,28 @@ def loadLabeledPoints(sc, path, minPartitions=None):
         minPartitions = minPartitions or min(sc.defaultParallelism, 2)
         return callMLlibFunc("loadLabeledPoints", sc, path, minPartitions)
 
+    @staticmethod
+    def appendBias(data):
+        """
+        Returns a new vector with `1.0` (bias) appended to
+        the end of the input vector.
+        """
+        vec = _convert_to_vector(data)
+        if isinstance(vec, SparseVector):
+            entries = dict(zip(vec.indices, vec.values))
```

Comment:
I think this will be inefficient.  Could you instead do:
```
newIndices = vec.indices + [len(vec)]
newValues = vec.values + [1.0]
return SparseVector(len(vec)+1, newIndices, newValues)
```